src/axom/spin/BVH.hpp

```diff
@@ -230,11 +253,53 @@ class BVH
    * \pre y != nullptr if dimension==2 || dimension==3
    * \pre z != nullptr if dimension==3
    */
-  void find( IndexType* offsets, IndexType* counts,
-             IndexType*& candidates, IndexType numPts,
-             const FloatType* x,
-             const FloatType* y,
-             const FloatType* z=nullptr ) const;
+  void findPoints( IndexType* offsets, IndexType* counts,
+                   IndexType*& candidates, IndexType numPts,
+                   const FloatType* x,
+                   const FloatType* y,
+                   const FloatType* z=nullptr ) const;
+
+  /*!
+   * \brief Finds the candidate bins that intersect the given rays.
```

Comment:
Just to clarify -- this function finds **all** boxes that could potentially intersect the ray, not just the candidates for the first intersection, right?